pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -403,6 +404,33 @@ export default class WindowsIntegrationManager implements IntegrationManager {
     );
   }
 
+  protected async syncHostFile() {
+    await Promise.all(
+      (await this.supportedDistros).map((distro) => {
+        return this.updateHostFile(distro.name);
+      }),
+    );
+  }
+
+  protected async updateHostFile(distro: string) {
```

Comment:
This whole function should be wrapped in a `try`/`catch`; if anything is thrown this breaks startup. (Doug just went through this in #4560 which is the only reason I know this…)